source/language/types.rst

```diff
@@ -368,54 +510,112 @@ their argument expression in parentheses:
-      | log      | ``float``                         | ``float``                            | Logarithm base :math:`e`               |
+      | log      | ``float``                         | ``float``                            | Logarithm base :math:`e`.              |
       +----------+-----------------------------------+--------------------------------------+----------------------------------------+
-      | mod      | ``int``, ``int``                  | ``int``                              |                                        |
-      |          |                                   |                                      |                                        |
-      |          | ``float``, (``int`` or ``float``) | ``float``                            |                                        |
+      | mod      | ``int``, ``int``                  | ``int``                              | Modulus.  The remainder from the       |
+      |          |                                   |                                      | integer division of the first argument |
+      |          | ``float``, ``float``              | ``float``                            | by the second argument.                |
       +----------+-----------------------------------+--------------------------------------+----------------------------------------+
-      | popcount | ``bit[_]``, ``uint``              | ``uint``                             |                                        |
+      | popcount | ``bit[_]``                        | ``uint``                             | Number of set (1) bits.                |
       +----------+-----------------------------------+--------------------------------------+----------------------------------------+
-      | pow      | ``int``, ``uint``                 | ``int``                              |                                        |
+      | pow      | ``int``, ``uint``                 | ``int``                              | :math:`\texttt{pow(a, b)} = a^b`.      |
       |          |                                   |                                      |                                        |
       |          | ``float``, ``float``              | ``float``                            | For floating-point and complex values, |
       |          |                                   |                                      | the principal value is returned.       |
       |          | ``complex``, ``complex``          | ``complex``                          |                                        |
       +----------+-----------------------------------+--------------------------------------+----------------------------------------+
-      | rotl     | ``bit[n]``, (``int`` or ``uint``) | ``bit[n]``                           |                                        |
+      | rotl     | ``bit[n]``, ``int``               | ``bit[n]``                           | Rotate the bits in the representation  |
```

Comment:
why drop `uint`?

Reply:
This table was written before the new implicit casting rules. `uint` autopromotes to `int` now if it's in range, and since these are unsized compile-time calls, that conversion will always succeed. I think explicitly saying "`int` or `uint`" here means we have to specify an order that the overloads are tried, but there wouldn't actually be any difference in them, so it seemed simpler to remove it and just use the implicit cast.

(I must have forgotten I did this, or I would have written it in the commit message.)

Reply:
Updated to uint and bit only in https://github.com/openqasm/openqasm/pull/366/commits/307fa094e8be792a845623101323999a452ab261